### PR TITLE
MonoOnlyFact -> Fact

### PR DESCRIPTION
**Pull request by Youssef1313**
Will see if the test will still pass or not.

### CODE REVIEW

**Review by RikkiGibson — APPROVED** [review]

**Comment by RikkiGibson** [comment]
Does this seem fine @jaredpar?

File: src/Compilers/Test/Core/Assert/ClrOnlyFactAttribute.cs [hunk 1]
```diff
@@ -79,15 +77,4 @@ private static string GetSkipReason(ClrOnlyReason reason)
             }
         }
     }
-
-    public sealed class MonoOnlyFactAttribute : FactAttribute
-    {
-        public MonoOnlyFactAttribute(string reason)
-        {
-            if (!MonoHelpers.IsRunningOnMono())
```
Comment (RikkiGibson) [inline src/Compilers/Test/Core/Assert/ClrOnlyFactAttribute.cs:87]: I imagine there are still other usages of this helper?

**Review by jcouv — APPROVED** [review]
LGTM Thanks (iteration 2)

File: src/Compilers/Test/Core/Assert/ClrOnlyFactAttribute.cs (hunk 1 above)
Comment (Youssef1313) [reply src/Compilers/Test/Core/Assert/ClrOnlyFactAttribute.cs:87]: Yeah there are multiple usages.

**Comment by RikkiGibson** [comment]
Thanks @Youssef1313!

**Pull request merged**